backend/users/views/team/user.py

```diff
@@ -0,0 +1,98 @@
+from typing import List, Optional
+
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from pydantic import BaseModel
+
+from paul.display.build_url import build_ngohub_url
+from paul.views.data_model import BasePageProps, Breadcrumb
+from users.models import User
+
+
+class Tab(BaseModel):
+    value: str
+    label: str
+
+
+class UserPageProps(BasePageProps):
+    tabs: list[Tab]
+    baseUrl: str
+
+    currentTab: str
+    tabTitle: str
+
+    errors: Optional[dict[str, list[str]]] = None
+
+
+PAGE_TABS = {
+    "info": {
+        "label": _("Info"),
+        "value": "info",
+    },
+    "role": {
+        "label": _("User Role"),
+        "value": "role",
+    },
+    "permissions": {
+        "label": _("Permissions"),
+        "value": "permissions",
+    },
+    "activity-log": {
+        "label": _("Activity Log"),
+        "value": "activity-log",
+    },
+}
+
+
+def get_requested_user(user_id: str) -> User:
```

Comment:
Because we retrieve the user by id, we could name this function in a more generic way. Maybe `get_user`?